cache/models/subject.js

```diff
@@ -39,6 +39,20 @@ const filters = {
 };
 
 /**
+ * Given absolutePath, return whether the subject is in cache
+ *
+ * @param {String} absolutePath
+ * @returns {Promise} resolves to true for found, false for not
+ */
+function isSubjectInCache(absolutePath) {
```

Comment:
boolean functions like this should take the form of a statement, not a question, so this function should be called "subjectIsInCache" or "subjectAlreadyExists" or "subjectExists" or "subjectInSampleStore"